src/components/WalletDropdown/MiniPortfolio/Activity/parseLocal.test.ts

```diff
@@ -10,23 +10,24 @@ import {
   TransactionDetails,
   TransactionType,
 } from 'state/transactions/types'
+import { renderHook } from 'test-utils'
 
-import { parseLocalActivity } from './parseLocal'
+import { parseLocalActivity, useLocalActivities } from './parseLocal'
 
 const oneUSDCRaw = '1000000'
 const oneDAIRaw = '1000000000000000000'
 
-function buildSwapInfo(
-  type: TradeType,
+function mockSwapInfo(
```

Comment:
var's have to be prefixed with "mock" in order to be used in then jest.mock fn scope below